setup_resources.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Load environment variables from .env file
+set -a
+source .env
+set +a
+
+# Set cluster-specific environment variables
+RESOURCEGROUP="v4-westeurope"
+DATABASE_ACCOUNT_NAME="${USER}-aro-${LOCATION}"
+KEYVAULT_PREFIX="${USER}-aro-${LOCATION}"
+
+# Create resource group
+echo "Ensuring resource group $RESOURCEGROUP in $LOCATION..."
```

Comment:
I am not sure about the flow/logic of this file usage but a small check of the az CLI exists and valid connectivity (e.g., running `az account show`) could improve user/developer experience. Feel free to ignore